Sources/Commands/PackageCommands/AddProduct.swift

```diff
@@ -42,7 +42,7 @@ extension SwiftPackageCommand {
         @Argument(help: "The name of the new product")
         var name: String
 
-        @Option(help: "The type of target to add, which can be one of 'executable', 'library', 'static-library', 'dynamic-library', or 'plugin'")
+        @Option(help: "The type of target to add.")
```

Comment:
nit: its not really standard whether doc strings end with a period `.` or not. Most don't seem to, so maybe we standardize on that.